tweaks.sh

```diff
@@ -67,7 +67,13 @@ while [[ $# -gt 0 ]]; do
           edit_firefox="true" ;;
       esac
 
-      if ! has_command firefox; then
+      if [[ -d "${FIREFOX_FLATPAK_DIR_HOME}" ]]; then
+          FIREFOX_DIR_HOME=${FIREFOX_FLATPAK_DIR_HOME}
+	  mkdir -p ${FIREFOX_FLATPAK_THEME_DIR}
```

Comment:
And, force making a folder in argument parsing?

Reply:
Where to put this? There is no folder "firefox-theme" on fresh install.

Reply:
Go explore `lib-core.sh` and `lib-install.sh`, and take a look the `install_firefox_theme` function. You better to put your code there.

Oh yeah, also, please take a look at `install_dash_to_dock_theme` and imitate the way we switch the correct path for Dash to Dock installation.

Anyways, we're currently freezing our repo for some refactoring. You can come back and recheck the code tomorrow